hasher-matcher-actioner/hmalib/lambdas/api/collabs.py

```diff
@@ -56,17 +62,19 @@ def get_available_schemas():
 
         A future version could pull from a static location or scan all packages
         for subclasses.
-
-        PS: Is schemata the correct plural?
         """
-        configs = tx_collab_config.get_all_collab_configs()
+        apis = tx_apis.ToggleableSignalExchangeAPIConfig.get_all()
```

Comment:
Why not prevent this from returning LocalFileSignalExchangeAPI?

Reply:
Fair. Moved it into `ToggleableSignalExchangeAPIConfig`